lib/creditsafe/client.rb

```diff
@@ -78,6 +81,8 @@ def handle_message_for_response(response)
     end
 
     def invoke_soap(message_type, message)
+      retries ||= RETRY_LIMIT
```

Comment:
The name feels a little counterintuitive as I'd expect it to be the number of retries already done. Feels like really this variable is more of an `attempts_remaining`, really?